evennia/contrib/actions/setup.py

```diff
@@ -0,0 +1,20 @@
+from evennia import ObjectDB
```

Comment:
@andrei-pmbcn 

Yes, this is called a circular import error and is solved by delaying the import. It's best to break the cycle in the utils.py modules to allow utils to easily be imported from the other modules.  Here is the recommended and efficient way to do a delayed import: 

```python
_ACTION_CHARACTER = None

def function_needing_actionchar():
    global _ACTION_CHARACTER
    if not _ACTION_CHARACTER:
        from evennia.contrib.actions.typeclasses import ActionCharacter as _ACTION_CHARACTER

    # use _ACTION_CHARACTER below ...
```